pyproject.toml

```diff
@@ -171,8 +171,13 @@ select = ["F", "E", "W", "B", "I", "N", "D", "C90", "UP"]
 ignore = [
     "B904",
     "E501",
-    "D1",    # Allow missing docstrings.
-    "D417",  # Don't require descriptions for all arguments.
```

Comment:
We're removing D417 here? That doesn't seem feasible

Reply:
I thought so too, but evidently this is only enforced if there's a proper `Arguments/Args/Parameters` section in the docstring.

So I think this rule doesn't really demand "function args need to be fully documented," it just demands "function args need to be fully documented or not at all...no half measures."